node-operators/validators/onboarding/opt-in-to-tanssi.md

```diff
@@ -0,0 +1,274 @@
+
+--8<-- 'text/node-operators/validators/onboarding/run-a-validator/prerequisites.md'
+
+## Opt-In to Tanssi-Enabled Vaults {: #opt-in-tanssi-vaults }
+
+Before enabling your operator to be an active Tanssi network validator, you must opt in to at least one of the Tanssi-enabled vaults. The following sections describe several ways to opt into the vaults.
+
+### Opt-In Using the Symbiotic CLI {: #opt-in-tanssi-vaults-with-cli }
+
+If you have correctly installed the [Symbiotic CLI](#setting-up-the-cli) and you want to sign the transaction using a ledger device, then run the following command, replacing the `VAULT_ADDRESS` with the specific one you want to join and `OPERATOR_ADDRESS` with your account:
+
+=== "Mainnet"
+
+    ```bash
+    python3 symb.py opt-in-vault VAULT_ADDRESS --ledger --ledger-account OPERATOR_ADDRESS
+    ```
+
+=== "Testnet (Sepolia)"
+
+    ```bash
+    python3 symb.py opt-in-vault {{ networks.symbiotic.contracts.sepolia.vault }} --ledger --ledger-account OPERATOR_ADDRESS --chain sepolia
+    ```
+
+If you want to sign the transaction directly using the operator's account private key, then run the following command, replacing the `PRIVATE_KEY` parameter:
+
+=== "Mainnet"
+
+    ```bash
+    python3 symb.py opt-in-vault VAULT_ADDRESS --private-key PRIVATE_KEY
+    ```
+
+=== "Testnet (Sepolia)"
+
+    ```bash
+    python3 symb.py opt-in-vault {{ networks.symbiotic.contracts.sepolia.vault }} --private-key PRIVATE_KEY --chain sepolia
+    ```
+
+!!! warning
+    Note that this method requires you to expose your private key; therefore, it is not recommended.
+
+### Opt-In Using Etherscan {: #opt-in-tanssi-vaults-with-etherscan }
+
+You can interact with Symbiotic's smart contracts using Etherscan, and sign the transaction using a browser wallet ([Metamask](/builders/toolkit/ethereum-api/wallets/metamask/#install-the-metamask-extension){target=\_blank}, for example). 
+
+To open the contrat's page, open the link:
+
+=== "Mainnet"
+
+    [https://etherscan.io/address/{{ networks.symbiotic.contracts.mainnet.opt_in_vault }}#writeContract](https://etherscan.io/address/{{ networks.symbiotic.contracts.mainnet.opt_in_vault }}#writeContract){target=\_blank}
+
+=== "Testnet (Sepolia)"
+
+    [https://sepolia.etherscan.io/address/{{ networks.symbiotic.contracts.sepolia.opt_in_vault }}#writeContract](https://sepolia.etherscan.io/address/{{ networks.symbiotic.contracts.sepolia.opt_in_vault }}#writeContract){target=\_blank}
+
+Click on `Connect to Web3`, and select your preferred wallet (e.g. Metamask):
+
+![Connect to Web3 step](/images/node-operators/validators/onboarding/opt-in-to-tanssi/opt-in-to-tanssi-1.webp)
+
+!!! note
+    You can configure Metamask to use a cold wallet.
+
+Once connected:
+
+1. Expand the `optin` function
+2. Insert the VAULT_ADDRESS
```

Comment:
```suggestion
2. Insert the `VAULT_ADDRESS`
```

Is VAULT_ADDRESS supposed to tie into a previous step? Why is it capitalized like that, I'm assuming it's a variable, which is why I put it in backticks in my suggestion. I just want to make sure it's going to be clear to users what they need to do here since it doesn't seem like they've set up a variable in this guide. If it's not a variable, might as well just do "Insert the vault address"